internal/config/help.go

```diff
@@ -18,11 +18,12 @@ var CLI = help{
 	title:   Title,
 	version: Version,
 	usage: map[int][]string{
-		0: {0: "--config", 1: "<file>", 2: "Specify config file"},
+		0: {0: "-c, --config", 1: "<file>", 2: "Specify config file"},
```

Comment:
does this still end up being aligned properly when you run `--help`?

i consider my code here for the CLI alignment to be pretty janky and it may be sensitive to this change

Reply:
I checked and it's aligned _in a way_ 🤔

Reply:
hmm yeah, i took a look - definitely busted with this change

i don't consider your change to be at fault here, I blame my janktastic code that surrounds it

with that being said; we'll have to refactor the help cli somehow to account for this

Reply:
I took a look, but as you said, that can of worms would be a nice use of the new flag (no offense). Might need a refactor (or perhaps outsource the whole CLI thing to a dependency)

Reply:
i'd like to avoid 3rd party dependencies, but stdlib `flags` package would probably work just fine

to tell you the truth i never expected this much involvement (incredibly grateful to see it though), or else I wouldn't have set contributors up for failure in this context - so sorry about that!

if i have time i'll try to look into this, but if you just hack the thing to bits and replace it, I won't be at all offended

you can expect my usual annoying gopher style feedback, or I may collab with you and modify it - but that's not personal at all and I'll always accept feedback in the other direction :D

but with that in mind; go ahead and rip that shit up if you find the time before I do (open invitation to any contributor)